Python/Product/Analysis/Parsing/Tokenizer.cs

```diff
@@ -844,8 +845,11 @@ public Tokenizer(PythonLanguageVersion version, ErrorSink errorSink = null, Toke
         }
 
         public static bool IsIdentifierChar(char ch) {
-            if (IsIdentifierStartChar(ch)) {
-                return true;
+            // ASCII case
+            if (ch <= 7F) {
+                return ch <= 'Z'
+                    ? ch >= 'A' || ch >= '0' && ch <= '9'
+                    : ch >= 'a' && ch <= 'z' || ch == '_';
```

Comment:
(Lost the old comment)

Digits are correct here. Should we add the fast case (without digits) to `IsIdentifierStartChar` as well? Does that have an impact?

Reply:
In mutate std lib tests impact is small (about 1% average), but there is no big deal to add it.